ingestors/maven.go

```diff
@@ -36,25 +36,14 @@ func (ingestor *MavenIngestor) Schedule() string {
 }
 
 func (ingestor *MavenIngestor) Ingest() []data.PackageVersion {
-	bookmark, err := getBookmarkTime(ingestor, time.Now().AddDate(-7, 0, 0))
```

Comment:
I'm not sure if this was intended to be "7 days ago" or not, but it's actually "7 years ago"